packages/terra-tag/src/Tag.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Tag.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  TAB: 9,
```

Comment:
Is this the only keycode we want? [Button](https://github.com/cerner/terra-core/blob/Section-header/packages/terra-button/src/Button.jsx#L10-L12) supports 3

Reply:
We do not have active or active/focus themes for tags so the other two buttons i.e. space and enter are not needed for Tag.

Reply:
If there are no focus styling on clickage tabs, how will user know the tag is clickable? Is there a different styling for tags that have an onclick given no icon is added to indicate an action will be made.

Reply:
Yes we have separate styling for the tags which are clickable. The tags with no onClick are not going to be accessible.

Reply:
Can we add these examples to the terra-site index page examples then? It appears we have static tags and href tags only displayed

Reply:
I have static and clickable tags added on the terra-site page. 
![screenshot 5](https://user-images.githubusercontent.com/35706779/37003569-25fc237c-2093-11e8-894e-e0c1c9e5cf53.jpeg)
and also in the #/tests/tag-tests i have added all the possible scenarios.

Is there anywhere else I need to added these tests?

Reply:
Ah onClick got me. Looks good.